homeassistant/components/samsungtv/media_player.py

```diff
@@ -152,16 +156,37 @@ def update(self):
 
     def get_remote(self):
         """Create or return a remote control instance."""
+
+        # Try to find correct method automatically
+        if self._config["method"] not in METHODS:
+            for method in METHODS:
+                try:
+                    self._config["method"] = method
+                    LOGGER.debug("try config: %s", self._config)
+                    self._remote = self._remote_class(self._config.copy())
+                    self._state = STATE_ON
+                    LOGGER.info("found working config: %s", self._config)
+                    break
+                except Exception as err:
```

Comment:
Pylint complains about "too general exception", but at this point the exception doesn't matter. It's (currently) not working and we have to try it later. How to handle this?

Reply:
@MartinHjelmare what to do here? The concrete reason doesn't matter, we just need to know it's not working.

Reply:
We should catch a specific exception. What error are we expecting?